packages/components/organisms/f-footer/test/specs/component/shared/f-footer.component.spec.js

```diff
@@ -1,17 +1,46 @@
 const forEach = require('mocha-each');
+
+const { buildUrl } = require('@justeat/f-wdio-utils/src/storybook-extensions.js');
 const Footer = require('../../../../test-utils/component-objects/f-footer.component');
 
-const footer = new Footer();
+const footer = new Footer('organism', 'footer-component');
+
+function formatLocale (tenant) {
```

Comment:
Ditto.

Remove this function and just change the tests to the full tenant code